docs/package.json

```diff
@@ -1,12 +1,16 @@
 {
-    "scripts": {
-      "clean": "rm -rf node_modules yarn-error.log css static/css *.log *.sqlite build/ .extra templates",
-      "watch": "watch -p './**/*.md' -c 'yarn run build-css'",
-      "build": "documentation-builder --base-directory . --output-path templates --template-path template.html --tag-manager-code 'GTM-K92JCQ' --no-link-extensions --force",
-      "test": "",
-      "serve": "cd dist && FLASK_DEBUG=true FLASK_APP=app.py flask run -h 0.0.0.0 -p $PORT"
-    },
-    "dependencies": {
-      "watch-cli": "^0.2.2"
+  "scripts": {
+    "clean": "rm -rf node_modules yarn-error.log css static/css *.log *.sqlite build/ .extra templates",
+    "watch": "watch -p 'static/sass/**/*.scss' -p 'node_modules/vanilla-framework/scss/*.scss' -c 'yarn run build-css'",
+    "build": "yarn run build-css && yarn run build-docs",
+    "build-css": "node-sass static/sass --output static/css",
+    "build-docs": "documentation-builder --base-directory . --output-path templates --template-path template.html --tag-manager-code 'GTM-K92JCQ' --no-link-extensions --force",
+    "test": "",
+    "serve": "cd dist && FLASK_DEBUG=true FLASK_APP=app.py flask run -h 0.0.0.0 -p $PORT"
+  },
+  "dependencies": {
+    "node-sass": "^4.5.3",
+    "vanilla-framework": "1.7.0-beta-2.1",
```

Comment:
Why does vanilla-framework need a dependency on vanilla-framework? Couldn't we just use the local files?

Reply:
Hahahaha oh yeah

Reply:
I think the only decent option to manage that in the run script is to use `./run --node-module ../` because of the container sandboxing.
I think this would involve manually entering that each time, and it would have risks of being confused by running local dev changes that you don't expect.

When talking about recreating the run script, I want to make an optional config file which could have standard argument like this if we chose to.
Although if we changed it to use the parent run script with a `serve-docs` endpoint, the local vanilla files would be available in the container.

Reply:
From what @WillMoggridge says it sounds like this is something that will be possible at a later date? If that's the case I don't see any reason why this can't be merged?